examples/psf_wf_movie.py

```diff
@@ -0,0 +1,258 @@
+
+import warnings
+import numpy as np
+import galsim
+
+try:
+    import matplotlib.pyplot as plt
+    import matplotlib.animation as anim
+except ImportError:
+    raise ImportError("This demo requires matplotlib!")
+try:
+    from astropy.utils.console import ProgressBar
+except ImportError:
+    raise ImportError("This demo requires astropy!")
+
+def simple_moments(img):
+    """Compute unweighted 0th, 1st, and 2nd moments of image.  Return result as a dictionary.
+    """
+    array = img.array
+    scale = img.scale
+    x = y = np.arange(array.shape[0])*scale
+    y, x = np.meshgrid(y, x)
+    I0 = np.sum(array)
+    Ix = np.sum(x*array)/I0
+    Iy = np.sum(y*array)/I0
+    Ixx = np.sum((x-Ix)**2*array)/I0
+    Iyy = np.sum((y-Iy)**2*array)/I0
+    Ixy = np.sum((x-Ix)*(y-Iy)*array)/I0
+    return dict(I0=I0, Ix=Ix, Iy=Iy, Ixx=Ixx, Iyy=Iyy, Ixy=Ixy)
+
+def ellip(mom):
+    """Convert moments dictionary into dictionary with ellipticity (e1, e2) and size (rsqr).
+    """
+    rsqr = mom['Ixx'] + mom['Iyy']
+    return dict(rsqr=rsqr, e1=(mom['Ixx']-mom['Iyy'])/rsqr, e2=2*mom['Ixy']/rsqr)
+
+def make_movie(args):
+    """Actually make the movie of the atmosphere given command line arguments stored in `args`.
+    """
+
+    # Initiate some GalSim random number generators.
+    rng = galsim.BaseDeviate(args.seed)
+    u = galsim.UniformDeviate(rng)
+
+    # The GalSim atmospheric simulation code describes turbulence in the 3D atmosphere as a series
+    # of 2D turbulent screens.  The galsim.Atmosphere() helper function is useful for constructing
+    # this screen list.
+
+    # First, we estimate a weight for each screen, so that the turbulence is dominated by the lower
+    # layers.  For this, we just interpolate the weights given by Jee&Tyson (2011) onto a set of
+    # uniformly-spaced layers between 0 and 15.46 km altitude.
+    JT_alts = [0.0, 2.58, 5.16, 7.73, 12.89, 15.46]  # km
+    JT_weights = [0.652, 0.172, 0.055, 0.025, 0.074, 0.022]  # weight
+    JT_interp = galsim.LookupTable(JT_alts, JT_weights, interpolant='linear')
+
+    alts = 15.46*np.arange(args.nlayers)/(args.nlayers-1)  # uniformly spaced altitudes in km.
+    weights = JT_interp(alts)  # interpolate the weights
+    weights /= sum(weights)  # and renormalize
+
+    # Each layer can have its own turbulence strength (roughly inversely proportional to the Fried
+    # parameter r0), wind speed, wind direction, altitude, and even size and scale (though note that
+    # the size of each screen is actually made infinite by "wrapping" the edges of the screen.)  The
+    # galsim.Atmosphere helper function is useful for constructing this list, and requires lists of
+    # parameters for the different layers.
+
+    spd = []  # Wind speed in m/s
+    dirn = [] # Wind direction in radians
+    r0_500 = [] # Fried parameter in m at a wavelength of 500 nm.
+    for i in xrange(args.nlayers):
+        spd.append(u()*20.0)  # Use a random speed between 0 and 20 m/s for each layer
```

Comment:
Could we perhaps define ```max_speed=20``` elsewhere in the script, like above this loop, and then use ```max_speed``` here?